### DISCUSSION TITLE
V5.0版本，图形编辑器下面的流程、脑图、拓扑三项都报错了

### DISCUSSION THREAD

**Discussion by yshaoliang** [ant-design/ant-design-pro#9222] [Q&A]
Unhandled Rejection (TypeError): Cannot read properties of undefined (reading 'Canvas')

▼ 18 stack frames were expanded.
_
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14518:19
e
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14454:19
_
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14752:15
e
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14454:19
266
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14490:18
(anonymous function)
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14491:4
(anonymous function)
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14442:28
./node_modules/_@antv_g6@2.2.6@@antv/g6/build/plugin.tool.minimap.js
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:14443:2
__webpack_require__
http://localhost:8000/mf-va_remoteEntry.js:23:42
Module../node_modules/_gg-editor@2.0.4@gg-editor/es/components/Minimap/index.js
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:18316:1
__webpack_require__
http://localhost:8000/mf-va_remoteEntry.js:23:42
Module../node_modules/_gg-editor@2.0.4@gg-editor/es/index.js
http://localhost:8000/mf-dep_vendors-node_modules__gg-editor_2_0_4_gg-editor_es_index_js.fd61fd03.async.js:18899:77
__webpack_require__
http://localhost:8000/mf-va_remoteEntry.js:23:42
Module../src/.umi/.cache/.mfsu/mf-va_gg-editor.js
http://localhost:8000/mf-dep_src_umi_cache_mfsu_mf-va_gg-editor_js.350c8fb5.async.js:40:67
__webpack_require__
http://localhost:8000/mf-va_remoteEntry.js:23:42
(anonymous function)
http://localhost:8000/mf-va_remoteEntry.js:672:719
__webpack_require__.m.<computed>
webpack:/ant-design-pro/webpack/runtime/remotes loading:1369
options.factory
webpack:/ant-design-pro/webpack/runtime/react refresh:6
▲ 18 stack frames were expanded.
__webpack_require__
.ant-design-pro/webpack/bootstrap:24
  21 | 	var execOptions = { id: moduleId, module: module, factory: __webpack_modules__[moduleId], require: __webpack_require__ };
  22 | 	__webpack_require__.i.forEach(function(handler) { handler(execOptions); });
  23 | 	module = execOptions.module;
> 24 | 	execOptions.factory.call(module.exports, module, module.exports, execOptions.require);
     | ^  25 | } catch(e) {
  26 | 	module.error = e;
  27 | 	throw e;

▼ 3 stack frames were expanded.
fn
webpack:/ant-design-pro/webpack/runtime/hot module replacement:61
Function.__webpack_require__.t
webpack:/ant-design-pro/webpack/runtime/create fake namespace object:10
async
http://localhost:8000/p__editor__flow.js:382:5

**Comment by LukerSpringtree** [comment]
@yshaoliang 
你好我想问一下, 现在怎么样才能下载到
图形编辑器 相关的代码?
能帮忙搞一份吗?
